core/sdk-core/src/main/java/software/amazon/awssdk/core/retry/RetryPolicy.java

```diff
@@ -79,12 +67,27 @@ public Integer numRetries() {
         return numRetries;
     }
 
+    public Builder toBuilder() {
+        return builder().numRetries(numRetries).retryCondition(retryConditionFromBuilder).backoffStrategy(backoffStrategy);
+    }
+
     public static Builder builder() {
         return new Builder();
     }
 
-    public Builder toBuilder() {
-        return builder().numRetries(numRetries).retryCondition(retryConditionFromBuilder).backoffStrategy(backoffStrategy);
+    public static RetryPolicy defaultRetryPolicy() {
+        return RetryPolicy.builder()
+                          .backoffStrategy(BackoffStrategy.defaultStrategy())
+                          .numRetries(SdkDefaultRetrySetting.DEFAULT_MAX_RETRIES)
+                          .retryCondition(RetryCondition.defaultRetryCondition())
+                          .build();
+    }
+
+    public static RetryPolicy noRetryPolicy() {
+        return RetryPolicy.builder()
+                          .backoffStrategy(BackoffStrategy.none())
+                          .retryCondition(RetryCondition.noRetryCondition())
```

Comment:
Some inconsistencies here between `none()` and `noXXX()`. I'd vote for `RetryCondition.none()`

Reply:
Will change to none.